tests/v2/validation/provisioning/rke2/etcd_backup_restore_test.go

```diff
@@ -406,7 +407,13 @@ func (r *RKE2EtcdSnapshotRestoreTestSuite) watchAndWaitForPods(client *rancher.C
 		}
 		isIngressControllerPodPresent := false
 		isKubeControllerManagerPresent := false
+		foundPendingPod := false
+		var podErrors []error
+
 		for _, pod := range pods.Data {
+			if pod.Namespace == "cattle-system" && strings.HasPrefix(pod.Name, "helm-operation") {
```

Comment:
Can you put `cattle-system` into the `const` block above? You can define `cattle-system` into a variable such as `cattleSystem` and then define it here.

Same with `helm-operation`, so that you're not having the value hardcoded.

Reply:
Done